fabrictestbed_extensions/fablib/node.py

```diff
@@ -1126,16 +1133,16 @@ def execute_thread(
 
     def execute(
         self,
-        command,
-        retry=3,
-        retry_interval=10,
-        username=None,
-        private_key_file=None,
-        private_key_passphrase=None,
-        quiet=False,
-        read_timeout=10,
+        command: str,
+        retry: int = 3,
+        retry_interval: int = 10,
+        username: str = None,
+        private_key_file: str = None,
+        private_key_passphrase: str = None,
+        quiet: bool = False,
+        read_timeout: int = 10,
         timeout=None,
-        output_file=None,
+        output_file: str = None,
```

Comment:
Maybe `typing.Union[str, os.PathLike]` (or maybe `pathlib.Path`) is more appropriate for file paths?